src/Avalonia.Controls/TopLevel.cs

```diff
@@ -232,6 +235,18 @@ protected bool AutoSizing
         }
 
         /// <inheritdoc/>
+        IRenderTarget IRenderRoot.CreateRenderTarget()
+        {
+            return _renderInterface.CreateRenderer(PlatformImpl.Handle);
```

Comment:
May be we should move `CreateRenderer` to `ITopLevelImpl`. Windowing platform has more information for negotiations with rendering platform than we do in `TopLevel` class